README.md

```diff
@@ -264,7 +244,7 @@ Available targets:
   travis/docker-tag-and-push          Tag & Push according Travis environment variables
 
 ```
-## Extending `build-harness` with targets from another repo
+# Extending `build-harness` with targets from another repo
```

Comment:
Why do you think it wants this to be an H1?

Reply:
(because the following 2 sections `## Share the Love` and `## References` are not subsections of this.)

Reply:
Exactly due to this

`first-line-heading/first-line-h1`

> File:[docs/terraform.md]
> ERROR! Found errors in [markdownlint] linter!
> ERROR:[docs/terraform.md:1 MD041/first-line-heading/first-line-h1 First line in file should be a top level heading >? [Context: "## Requirements"]]

https://github.com/cloudposse/.github/runs/865479515